Apps/StressTest/StressTest/ViewController.swift

```diff
@@ -290,7 +290,7 @@ class ViewController: UIViewController {
         print("Creating snapshotter")
         let snapshotter = Snapshotter(options: options)
         snapshotter.style.uri = .light
-        snapshotter.camera = mapView.cameraOptions
+        snapshotter.setCamera(to: CameraOptions(cameraState: mapView.cameraState))
```

Comment:
I wonder if we're going to need a shorthand for this.

Reply:
Potentially.. We do need to do some communication around `CameraState` vs `CameraOptions`.